### PR TITLE
KTL-1399 Align kotlin-metadata-jvm documentation releases with Kotlin releases

### PR DESCRIPTION
**Pull request by nikpachoo**
https://youtrack.jetbrains.com/issue/KTL-1399/Align-kotlin-metadata-jvm-documentation-releases-with-Kotlin-releases

### CODE REVIEW

**Comment by nikpachoo** [comment]
Not ready yet. The version name has a snapshot suffix

Screenshot 2024-06-07 at 13 50 23

**Comment by nikpachoo** [comment]
Fixed:
Screenshot 2024-06-17 at 11 58 06

File: .teamcity/builds/apiReferences/kotlinx/metadataJvm/KotlinxMetadataJvmBuildApiReference.kt [hunk 1]
```diff
@@ -81,3 +53,11 @@ object KotlinxMetadataJvmBuildApiReference : BuildType({
         dependsOnDokkaTemplate(KotlinxMetadataJvmPrepareDokkaTemplates, "$LIB_DIR/dokka-templates")
     }
 })
+
+fun removeVPrefix(input: String): String {
+    return if (input.startsWith("v")) {
```
Comment (zoobestik) [inline .teamcity/builds/apiReferences/kotlinx/metadataJvm/KotlinxMetadataJvmBuildApiReference.kt:58]: 🤔 is it necessary? just `return input.removePrefix("v")` doesn't work?

**Review by zoobestik — APPROVED** [review]

File: .teamcity/builds/apiReferences/kotlinx/metadataJvm/KotlinxMetadataJvmBuildApiReference.kt (hunk 1 above)
Comment (nikpachoo) [reply .teamcity/builds/apiReferences/kotlinx/metadataJvm/KotlinxMetadataJvmBuildApiReference.kt]: thanks, fixed.

**Pull request merged**